src/FlowtideDotNet.Substrait/Sql/Internal/SqlSubstraitVisitor.cs

```diff
@@ -83,39 +120,135 @@ public SqlSubstraitVisitor(SqlPlanBuilder sqlPlanBuilder, SqlFunctionRegister sq
             Debug.Assert(relationData != null);
 
             bool isBuffered = false;
+            bool isDistributed = false;
+            Expressions.FieldReference? scatterField = default;
+            int? partitionCount = default;
             if (createView.WithOptions != null)
             {
                 foreach(var opt in createView.WithOptions)
                 {
                     var upperName = opt.Name.ToString().ToUpper();
-                    if (upperName == "BUFFERED")
+                    if (upperName == SqlTextResources.Buffered)
                     {
                         var upperVal = opt.Value.ToSql().ToUpper();
                         if (upperVal == "TRUE")
                         {
                             isBuffered = true;
                         }
```

Comment:
```suggestion
                        var val = opt.Value.ToSql();
                        if (string.Equals(val, bool.TrueString, StringComparison.OrdinalIgnoreCase))
                        {
                            isBuffered = true;
                        }
```
Minor but could be done like that if wanting to avoid hardcoded strings, not sure if better :P